c/parallel/src/util/types.cpp

```diff
@@ -0,0 +1,47 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of CUDA Experimental in CUDA C++ Core Libraries,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+#include "types.h"
+
+#include <cuda/std/cstdint>
+
+#include "errors.h"
+#include <nvJitLink.h>
+#include <nvrtc.h>
+
+std::string_view cccl_type_enum_to_string(cccl_type_enum type)
```

Comment:
I don't think you need to reintroduce this. I removed it because we already have `cccl_type_enum_to_name` in `types.h` which does the same thing (see https://github.com/NVIDIA/cccl/pull/3763#discussion_r1956871963 and https://github.com/NVIDIA/cccl/pull/3763#discussion_r1953653234). You should be able to replace this function and things should still work fine.

Reply:
Removed. Thank you @NaderAlAwar